foundation/foundation-upgrade/src/test/resources/specifications/v1_10_0/spark-pipeline-messaging-migration.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Migrate a Spark pipeline module with the new CDI classes to ensure messaging compatibility with Java 17
+
+  Scenario: A Spark pipeline module with smallrye kafka is migrated to include the aissemble kafka messaging dependency and kafka CDI context
+    Given a POM with the "jar" packaging type
+    And a "smallrye-reactive-messaging-kafka" dependency
+    When the Spark Pipeline Messaging migration executes
+    Then the aissemble kafka messaging dependency is added to the POM
+    And the CDI container factory is updated with the messaging and kafka context objects
+    And the pipeline CDI context has the unecessary Kafka beans and imports removed
```

Comment:
S: I'd skip this bit.  It's just cleanup, and there's a chance this could cause a compile error that would be difficult to get out of without just disabling the migration. And if you had to disable it on the first pipeline in your project, subsequent pipelines wouldn't get the automated migration.

Reply:
Updated

Reply:
Verified this will not cause any issues leaving the extra beans